sdk/java-sdk-spring/src/main/scala/kalix/javasdk/impl/reflection/Reflect.scala

```diff
@@ -50,4 +54,32 @@ object Reflect {
     }
 
   }
+
+  private implicit val stringArrayOrdering: Ordering[Array[String]] =
+    Ordering.fromLessThan(util.Arrays.compare[String](_, _) < 0)
+
+  implicit val methodOrdering: Ordering[Method] =
+    Ordering.by((m: Method) => (m.getName, m.getReturnType.getName, m.getParameterTypes.map(_.getName)))
+
+  def lookupComponentClientField(instance: Any): List[ComponentClient] = {
```

Comment:
This is the main utility method being added. 
For the moment in use when receiving Action commands, but not yet for Workflow